requirements-freeze.txt

```diff
@@ -10,7 +10,7 @@ chardet==3.0.4
 -e git+https://github.com/ckan/ckanext-harvest.git@13dbb1eea428e6b274f98c36bf29d42e62a70af7#egg=ckanext_harvest
 -e git+https://github.com/GSA/ckanext-saml2.git@0e1f11cb3b211e6d337b86ffe7c07620821f7d7d#egg=ckanext_saml2
 -e git+https://github.com/GSA/USMetadata.git@fc26985504f9b14e92d0ed6d8387326e7fb6864d#egg=ckanext_usmetadata
-ckanext-dcat-usmetadata==0.1.10
+ckanext-dcat-usmetadata==0.1.11
```

Comment:
Shouldn't this also include a change to requirements.txt?

Reply:
@adborden yes, it should. Can you clarify please, if we should update master branch as well so any future deployments won't break production environment?

Reply:
Once the form is ready to ship, we'll [update requirements.txt on master](https://github.com/GSA/inventory-app/blob/master/requirements.txt#L8) to specify `ckanext-dcat-usmetadata~=0.1.11`. Then we run [make update-dependencies requirements](https://github.com/GSA/inventory-app#update-dependencies) to generate the new requirements-freeze.txt. That will pull in the latest compatible version of ckanext-dcat-usmetadata from pypi.

Reply:
Until we're ready to ship, `master` can stay as is.